mlflow/models/flavor_backend.py

```diff
@@ -67,12 +67,26 @@ def prepare_env(self, model_uri, capture_output=False):
 
     @abstractmethod
     def build_image(
-        self, model_uri, image_name, install_mlflow, mlflow_home, enable_mlserver, base_image=None
+        self,
+        model_uri,
+        output_dir,
```

Comment:
do we need to add `output_dir`?

https://github.com/mlflow/mlflow/blob/e10efeecd00ae200152d627f705aed4e3a8c1c41/mlflow/pyfunc/backend.py#L365-L370